artemis/modules/wordpress_plugins.py

```diff
@@ -186,6 +173,29 @@ def __init__(self, *args: Any, **kwargs: Any):
         with open(os.path.join(os.path.dirname(__file__), "data", "wordpress_plugin_readme_file_names.txt")) as f:
             self._readme_file_names = json.load(f)
 
+    def _get_plugins_from_homepage(self, url: str) -> List[Dict[str, Any]]:
+        links = get_links_and_resources_on_same_domain(url)
+
+        plugin_data = []
+        for link in links:
+            pattern = r"\/wp-content\/plugins\/([a-z-0-9]*)\/"
+            match = re.search(pattern, link)
+            if match:
+                slug = match.group(1)
+                data = json.loads(
+                    self.cached_get(f"https://api.wordpress.org/plugins/info/1.0/{slug}.json", "version-" + slug),
+                    timeout=3600,
```

Comment:
Isn't this timeout too big? 1 hour timeout seems over necessary

Reply:
I don't think plugin will get updated that frequently and I don't want to get banned on wp api